schemas/preferences.sublime-settings.json

```diff
@@ -1137,7 +1137,11 @@
         },
         "sublime_merge_path": {
           "description": "The path to the Sublime Merge executable. This should only be set if Sublime Merge is not installed in a standard location, such as when using a portable install.",
-          "type": "string",
+          "type": ["null", "string"],
+          "enum": [null, ""],
+          "markdownEnumDescriptions": [
+            "This will hide Sublime Merge related integrations like context/side bar menu items etc."
```

Comment:
Sorry but minor nit pick: Those descriptions are shown in a very clear context so no need for `This will`. It's clear that it refers to the `null` value. So can be just `Hides Sublime Merge...`.